Makefile

```diff
@@ -152,10 +152,12 @@ LINUX_CMAKE_PARAMS = \
 clean: clean-android clean-osx clean-ios clean-rpi clean-tests clean-xcode clean-linux
 
 clean-android:
-	@cd android/ && \
-	./gradlew clean
 	rm -rf ${ANDROID_BUILD_DIR}
+	rm -rf android/build
+	rm -rf android/tangram/build
 	rm -rf android/tangram/libs
+	rm -rf android/demo/build
+	rm -rf android/demo/libs
```

Comment:
Turns out the CMake configuration for Android native debugging is broken now that we've upgraded to 3.0 (due to this policy https://cmake.org/cmake/help/v3.3/policy/CMP0026.html). 

Since we effectively don't have an android-debug target right now, I'm going to leave that as a separate issue and merge this as-is.